bbb-common-message/src/main/scala/org/bigbluebutton/common2/msgs/PluginMsgs.scala

```diff
@@ -60,3 +60,17 @@ case class PluginLearningAnalyticsDashboardSendGenericDataMsgBody(
                                                             pluginName: String,
                                                             genericDataForLearningAnalyticsDashboard: PluginLearningAnalyticsDashboardGenericData
                                         )
+object PluginPersistEventMsg { val NAME = "PluginPersistEventMsg" }

```

Comment:
I think the name of this message should be `PluginPersistEventReqMsg`.
But I know that you just followed the pattern of the similar messages sent by plugins.

We need to take note to refactor the name of these messages in a subsequent PR, including `ReqMsg` at the end of all.

Reply:
Thanks for the suggestion. I already opened an issue to address this naming refactor: https://github.com/bigbluebutton/bigbluebutton/issues/22034